test/e2e/workloads/nsenter/daemonset.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: &name nsenter
+  labels:
+    app: *name
+spec:
+  selector:
+    matchLabels:
+      app: *name
+  template:
+    metadata:
+      labels:
+        app: *name
+    spec:
+      hostNetwork: true
+      hostPID: true
+      containers:
+      - image: alexeldeib/nsenter:latest
```

Comment:
there's no more "official" image for this? Should we store one on mcr?

Reply:
I don't know of a more official image, the existing nsenter ones are all random people. I built this over the weekend to be as minimal as possible, I'm happy to put it in MCR (especially since docker hub will get rate limited). https://github.com/alexeldeib/nsenter/blob/main/Dockerfile

It's a scratch dockerfile with static sleep + nsenter binaries. < 2MB for the whole image 🙂

Reply:
most common images like ubuntu etc have nsenter and sleep installed, but are pretty beefy, and slimmer stuff like distroless usually doesn't have the binaries.

Reply:
better?